lib/adapters/http.js

```diff
@@ -273,7 +276,7 @@ module.exports = function httpAdapter(config) {
 
     // Handle errors
     req.on('error', function handleRequestError(err) {
-      if (req.aborted && err.code !== 'ERR_FR_TOO_MANY_REDIRECTS') return;
+      if (req.aborted && err.code !== 'ERR_FR_TOO_MANY_REDIRECTS') reject(err);
```

Comment:
Without this request will never complete.

Reply:
just to expand more on this change, I haven't used `enhanceError` since I expect this will be an error thrown from client application so we return it as is. 
It's a bit strange that no one before run into the issue of stuck request